context/rootfs/scripts/init-registry.sh

```diff
@@ -29,34 +30,56 @@ certs_dir="$rootfs/certs"
 
 mkdir -p "$VOLUME" || true
 
+imageServer="nerdctl"
+if [[ $(ls ../cri/docker*.tar.gz) ]]; then
+    imageServer="docker"
+fi
+
 startRegistry() {
   n=1
   while ((n <= 3)); do
     echo "attempt to start registry"
-    nerdctl start $container && break
-    ((n++))
+    $imageServer start $container; break || ((n++))
     sleep 3
   done
 }
 
-check_registry() {
+docker_check_registry() {
```

Comment:
I am some kind of confused that, we use different ways  to define functions, like startRegistry, check_registry. We could use unified ones, all the `_` ways or  all the camel ways. WDYT?